ui/pages/import-token/import-token.component.js

```diff
@@ -281,7 +285,7 @@ class ImportToken extends Component {
     const isMainnetNetwork = this.props.chainId === '0x1';
 
     let standard;
-    if (addressIsValid && process.env.COLLECTIBLES_V1) {
```

Comment:
Is removing this feature flag intentional?

Reply:
Yes, the function used here is what gets the token standard, in order to get it to trigger for every added tokens I had to remove the feature flag